lib/widgets/main_menu.dart

```diff
@@ -44,167 +35,216 @@ class _MainMenuState extends State<MainMenu> {
     );
   }
 
+  List<Widget> _buildMenuElements(BuildContext context) {
```

Comment:
It seems that the method `_buildMenuElements` was put back here in this PR.
My view is that placing the `_buildxxxxxxxx(context)` methods directly into the ListView is a much clearer style (as is in the current master branch).

Reply:
Yes, I agree it’s much cleaner to keep this in the ListView directly, but because we have many conditional statements, I thought it’s better to keep it as a separate method.

Reply:
That makes sense. 
In future implementation, keeping using `_buildMenuElements` will make things easier.